libs/api/domains/vehicles/src/lib/api-domains-vehicles.service.ts

```diff
@@ -70,6 +72,22 @@ export class VehiclesService {
     return this.mileageReadingApi.withMiddleware(new AuthMiddleware(auth))
   }
 
+  async getVehiclesListV2(
+    auth: User,
+    input: CurrentvehicleswithmileageandinspGetRequest,
+  ): Promise<CurrentVehiclesWithMilageAndNextInspDtoListPagedResponse> {
+    return await this.getVehiclesWithAuth(
+      auth,
+    ).currentvehicleswithmileageandinspGet({
+      ...input,
+      permno: input.permno
+        ? input.permno.length < 5
+          ? `${input.permno}*`
+          : `${input.permno}`
+        : undefined,
```

Comment:
spooky

Reply:
its for wildcard search. 5 and over will be used for direct search. 1-5 wildcard. otherwise undefined resulting in no 
permno query sent.